cloudant-http/src/main/java/com/cloudant/http/interceptors/RequestLimitInterceptor.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright (c) 2016 IBM Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+ * except in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the
+ * License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific language governing permissions
+ * and limitations under the License.
+ */
+
+package com.cloudant.http.interceptors;
+
+import com.cloudant.http.HttpConnectionInterceptorContext;
+import com.cloudant.http.HttpConnectionResponseInterceptor;
+
+import org.apache.commons.io.IOUtils;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.HttpURLConnection;
+import java.util.concurrent.TimeUnit;
+import java.util.logging.Logger;
+
+public class RequestLimitInterceptor implements HttpConnectionResponseInterceptor {
+
+    private static final Logger logger = Logger.getLogger(RequestLimitInterceptor.class.getName());
+    private static int initialSleep = 250; // Default to 250 ms initial backoff time
```

Comment:
It is not intended to be configurable. It wasn't made final so it could be reduced (via reflection) during testing to avoid having the tests run long.